solidity/scripts/lcl-initialize.js

```diff
@@ -76,17 +76,32 @@ module.exports = async function () {
     }
 
     try {
-      await bondedECDSAKeepFactory.createSortitionPool(application)
-      console.log(`created sortition pool for application: [${application}]`)
+      console.log(`getting sortition pool for application [${application}]`)
 
       sortitionPoolAddress = await bondedECDSAKeepFactory.getSortitionPool(
         application
       )
     } catch (err) {
-      console.error("failed to create sortition pool", err)
-      process.exit(1)
+      console.error("failed to get sortition pool", err, "; creating a new one")
```

Comment:
Should we do `process.exit(1)` in the next line?

Reply:
But we don't :)

Reply:
So shouldn't we?

Reply:
Nope, if we catch an error from `getSortitionPool` we should try to create a new one instead of exit.

Reply:
🤦 Please forget